src/message.py

```diff
@@ -2,6 +2,10 @@ class Message:
     def display_welcome_message(self):
         print("Welcome to Tic Tac Toe")
 
+    def print_rules(self):
+        rules = """\nPlay this game by taking turns marking the board. \nWhen prompted, type your a number between 1 and 9 and press enter. \nIf that spot is taken, the computer will prompt you for a different spot. \nThe first player who gets three of their marks in a row wins! \nIf the board is full and neither player has three in a row, it is a draw and the game is over. \nAt the end of every game, you will have the option to play again or to exit.\n"""
+        print(rules)
```

Comment:
I just noticed that this class is actually calling `print()`, which I assume prints to a console.  Now that you have a ConsoleUI class (`UserInterface`), printing to the console should probably be owned by that class.  Instead the `Message` class methods should simply return the messages.

Reply:
This gets into a class doing 1 thing and doing it well. aka, Single Responsibility Principle. 

For the `Message` class, that one thing is returning message strings.
For the `UserInterface` class that one thing is interacting with the console.